src/thx/Arrays.hx

```diff
@@ -1243,7 +1252,27 @@ Finds the min element of the array given the specified ordering.
 
     return r;
   }
-
+/**
+  Pads out to len, ignores if len is less than Array length.
+**/
+  static public function pad<T>(arr:Array<T>,len:Int):Array<T>{
```

Comment:
It might be handy to let the caller pass in a value of type `T` to pad with, rather than assuming `null`.